DTS-STN/Service-Canada-Labs
# Design review changes

# Design review changes

This PR addresses the following tasks:

[Update card border and shadow](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=130417)

Before:

![Screenshot 2023-07-07 at 8 58 33 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/8cdb4772-5906-4161-800f-01cc8e648c9e)

After:

![Screenshot 2023-07-07 at 8 59 04 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/7378e2f5-c4dc-44c4-b569-880fbdf18588)

[Fix SCDS CallToAction (CTA) styles to match design](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=130483)

Before:

![Screenshot 2023-07-07 at 8 57 29 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/92c2fa1a-a612-45e3-96d4-cd9b0648db77)

After:

![Screenshot 2023-07-07 at 8 57 57 AM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/688e6ef2-ad4d-4feb-9409-a9eee2491e5a)


## Test Instructions

1. On homepage, see that card border and shadow matches what is found in the design [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=14229-32437&mode=design&t=ljFufD8m1TYwPRbE-0)
2. CTA on OAS page should match the appearance of the component [here](https://dts-stn.github.io/Service-Canada-Design-System/release/?path=/story/components-cta--default)

## Definition of Done
